src/harness-manager/templates/config.yaml

```diff
@@ -292,7 +292,7 @@ data:
       },
       {
         "name": "DeletedEntityIterator",
-        "enabled": true,
+        "enabled": false,
```

Comment:
@Raghav-Murali  could you please check if this change is necessary ? 

Also please review any other iterators which can cause any issues esp data loss

Reply:
@akshay-khole - These 2 Iterators are the ones that delete entries from account related collections. So disabling them for SMP makes sense.

We would have to look into the license_expiry cache that is currently being used in the License_Expiry iterator which might contain stale data which can inadvertently cause accounts to be marked for deletion. Will discuss about this with the PLG team.

Reply:
We should make this an external value.  These charts will be used for SaaS deployments soon and we will need the ability to maintain this capability.